src/main/perl/ComponentProxy.pm

```diff
@@ -236,15 +236,21 @@ sub _initialize
     if ($tree->{version}) {
         # version is part of the PMpost maven template
         # be aware of obscure tainting errors (version uses XS); untaint if needed
-        local $@;
-        eval {
-            $self->{VERSION_CONFIG} = version->new($tree->{version});
-        };
-        if ($@) {
-            $self->error("component $name has invalid version from config $tree->{version} (bug in profile?): $@");
-            return;
+        my $errmsg = "component $name has invalid version from config $tree->{version} (bug in profile?)";
+        if ($tree->{version} =~ m/^(v?\d+(?:\.\d+(?:\.\d+)))$/) {
+            local $@;
+            eval {
+                $self->{VERSION_CONFIG} = version->new($1);
+            };
+            if ($@) {
+                $self->error("$errmsg: $@");
+                return;
+            } else {
+                $self->verbose("component $name version from config $self->{VERSION_CONFIG}");
+            }
         } else {
-            $self->verbose("component $name version from config $self->{VERSION_CONFIG}");
+            $self->error("$errmsg: does not match regex pattern");
```

Comment:
i can print the regex pattern, i hope everyone can read those :smile: (the `v` is optional)